### PR TITLE
remove typing in conda requirements for py>=3.6

### PR DESCRIPTION
**Pull request by fhoehle**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -28,7 +28,7 @@ requirements:
     - singledispatch >=3.4.0.3
     - six
     - scikit-learn >=0.18
-    - typing
+    - typing # [not py>=36]
```
Comment (xhochy) [inline recipe/meta.yaml]:
```suggestion
    - typing  # [py<36]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -28,7 +28,7 @@ requirements:
     - singledispatch >=3.4.0.3
     - six
     - scikit-learn >=0.18
-    - typing
+    - typing  # [not py>=36]
```
Comment (xhochy) [inline recipe/meta.yaml]:
```suggestion
    - typing  # [py<36]
```

**Comment by fhoehle** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (fhoehle) [reply recipe/meta.yaml]: done

**Pull request merged**